src/format/elf64/section.c

```diff
@@ -0,0 +1,296 @@
+/**
+ * @file src/format/elf64/section.c
+ *
+ * `section.c` functions and data structures for reading ELF64 section headers.
+ *
+ * @author H Paterson.
+ * @copyright BSL-1.0.
+ * @date May 2020.
+ */
+
+#include "format/elf64/section.h"
+#include "status.h"
+#include <stdio.h>
```

Comment:
Hosted C stdlib dependencies shouldn't be in `libprim`, except in the platform folder.